lib/commands/project/project.js

```diff
@@ -95,6 +95,20 @@ function create(name, options, cb) {
     cloneSkeleton(name, framework, targetDir, function(err) {
       if (err) { return cb(err); }
       emit('Project %s created in %s', name, targetDir);
+
+      findProjectFile(targetDir, null, function(err, projPath) {
+        var projectFile = fs.readFileSync(projPath, 'utf8');
+        var projectJson = JSON.parse(projectFile);
+
+        projectJson.devDependencies['z-schema'] = '^3.12.0';
```

Comment:
@mohsen1 the dependencies are being injected during the ```swagger project create``` because we wanted to avoid running ```npm install``` twice, so I'm not sure that this edge case is necessary. If this is the wrong approach, then I misunderstood you yesterday. Shouldn't be a hard fix though.